### ISSUE TITLE
Can't change theme or color scheme whatsoever

### ISSUE THREAD

**Issue by scottymartini** [morpheusthewhite/spicetify-themes#354]
First off, I know nothing about coding and therefore anything I can accomplish on my own is extremely limited. My only interest is in pimping out my Spotify interface alone.

That said, I stumbled and fumbled my way into figuring out how to switch between themes and color schemes last night and got it working properly by use of the config.ini. My favorite that I had tried was the Vaporwave with NewRetro color scheme so I was going to settle w that one after messing around w the Dribbblish theme.

I couldn't get that one to work and I probably did something super wrong while trying to get it to work because not only does my Spotify interface not utilize Dribbblish, but it utilizes some other theme that I don't know and a white/salmon color scheme.

Sorry this is getting long.

So I figured eh whatever I'll just switch the theme back to the Vaporwave and put the color scheme as NewRetro and update it and apply it. Nothing changed, still the white/salmon UI, looks like Flatten maybe. So I restored and set it back to vanilla Spotify, which worked, then tried again. Back to the white/salmon screen. Fast forward like three collective hours and I still can't figure out what's going wrong. I can either use the white version or the vanilla version and nothing else.

I realize this is probably hard to understand because I'm not knowledgable enough in this field to use specific jargon or even theorize as to where I might've done something wrong and locked myself into this loop.

Bottom line, I had it right through trial and error for a while last night, tried Dribbblish, likely messed something up in the process, and now I can't switch out of this seemingly random theme/scheme combination unless I restore and then only use vanilla Spotify. If there's a way for me to start over from ground zero and completely wipe all the spicetify info on my laptop and just try again without touching Dribbblish with a fifty foot pole then that'd be ideal. I'm obviously not intuitive enough to handle this presumably basic task.

Thanks!

**Comment by scottymartini** [comment]
Oh and I forgot to mention, I really have tried everything under the sun within my capabilities. I've read the spicetify -h, gone through everything that looked like it might work, read several of these issue threads, re-downloaded Spotify about twenty times while doing certain actions before and after doing so, everything. Basically I gave it an honest shot lol hopefully one of y'all either knows what I did to restrict myself to this one theme (that I don't even type in btw, it just defaults to it) or can tell me how I can start over entirely. Either or.

**Comment by physcrowley** [comment]
It is possibly linked to the new Spotify version that has been causing a simply blacked-out Spotify app when applying or restoring any theme (except the default one). There is a new version of Spicetify (v.2.0.1) that seemingly works with Spotify version 1.1.59, and you can check out how to get it going here :

 [this Issue in the Spicetify-CLI Repo](https://github.com/khanhas/spicetify-cli/issues/564#issuecomment-841354875) 

I have been able to restore Themes (I'm using WintergatanBlueprint) following those instructions, but they don't work quite as they used to because some of the Spotify UI elements have changed a bit.

**Comment by scottymartini** [comment]
No dice. I will say that I've only encountered a black screen upon startup a handful of times. My problem seems to be unique to me in the sense that I have no issue getting the spiced up message like many others. Rather, I don't get to pick which spice I get, which would be fine if the one I'm stuck with was my style. It's as if the config.ini menu where I type the theme and scheme I want next to the corresponding fields is somehow disconnected from the spicetify program entirely, which makes no sense but that's essentially the effect as far as I can see. I really am stuck with this white and reddish theme regardless of how many re-installations or different series of commands I type. I'll be using vanilla Spotify until y'all smart people can make sense of the unfortunately vague description I can offer lol

**Comment by scottymartini** [comment]
For context, I closed every window and quit every app and trashed everything: my spicetify data folder and spotify itself. Then I put in the command to download the new version of spicetify and I assume it worked because it said "spicetify --help" instead of "spicetify -h" like I was used to seeing. So I redownloaded spotify and no black screen, it was just vanilla spotify as usual. So I put in "spicetify backup apply" and got an OK for everything except I had this message at the bottom:

fatal open /Applications/Spotify.app/Contents/Resources/Apps/zlink/css/user.css: no such file or directory

I get that one a lot, maybe it's the root of the problem. Idk. So I opened up my old buddy old pal config.ini and my color scheme was still on NewRetro but my theme had been set back to SpicetifyDefault. So I opened spotify again to see what would happen and lo and behold, a black screen. It's weird I've seen that white theme so many times that when I see this black screen every now and then, it actually feels like progress to me lol prob only me who sees it that way.

So I re-installed spotify because I tried "spicetify backup" after seeing the black screen and that's what it told me to do. I tried the same command and it gave me the "everything is ready" message, which I obviously don't believe anymore lol so then I went back to the config.ini and replaced "SpicetifyDefault" with "Vaporwave" and left the "NewRetro" entry alone bc it was in its right place next to color_scheme. After that, tried "Spicetify update" and got the same thing:

fatal open /Applications/Spotify.app/Contents/Resources/Apps/zlink/css/user.css: no such file or directory

Finally, I tried "spicetify apply" after that for the heck of it, got everything saying OK again, and got the "spotify is spiced up!" message. Spotify opens, the crowd falls quiet, and BAM white theme with salmon colored accents out of nowhere yet again.

I lose lol I'll try any ideas y'all throw at me though

**Comment by scottymartini** [comment]
Oh last thing I swear. Lol I did remember to re-download all the themes to begin with and I did transfer them into the spicetify_data folder before replacing the default theme w the vaporwave one. So that's probably not the issue, I'm covering all my bases as far as I can tell. Sorry for the text wall

**Comment by khanhas** [comment]
Legacy Spotify themes are incompatible with new Spotify and nothing you can do about them.

**Comment by khanhas** [comment]
Also please take a look at release note to know what to expect in new version https://github.com/khanhas/spicetify-cli/discussions/560
